src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import cn from 'classnames';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  setFilter: (q: Filter) => void;
+  filterOption: Filter;
+  todos: Todo[];
+}
+
+export const Footer: React.FC<Props> = ({ setFilter, filterOption, todos }) => {
+  const completedTodo = todos.filter(todo => todo.completed).length;
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {
+          todosLeft === 1
+            ? `${todosLeft} item left`
```

Comment:
Best way it's create `pluralize` helper in separate file to such cases: https://stackoverflow.com/a/39835908